### PR TITLE
Add new function: omrsig_is_master_signal_handler

### PR DESCRIPTION
**Pull request by babsingh**
1. Add stubs for omrsig_is_master_signal_handler

	omrsig_is_master_signal_handler is a newly added function. It determines
	if a provided OS handler is a predefined master signal handler. 
	There are two master signal handlers: i) masterASynchSignalHandler is
	used for asynchronous signals and ii) masterSynchSignalHandler is used
	for synchronous signals.

2. Add Unix implementation of omrsig_is_master_signal_handler

Signed-off-by: Babneet Singh <sbabneet@ca.ibm.com>

### CODE REVIEW

**Comment by babsingh** [comment]
fyi - @pshipton @charliegracie

File: port/common/omrport.tdf [hunk 1]
```diff
@@ -1069,3 +1069,6 @@ TraceException=Trc_PRT_signal_mapOSSignalToPortLib_ERROR_unknown_signal Group=si
 TraceEntry=Trc_PRT_signal_omrsig_register_os_handler_entered Group=signal Overhead=1 Level=1 NoEnv Template="omrsig_register_os_handler: Entered, portLibrarySignalFlag=0x%X, newOSHandler=%p"
 TraceExit=Trc_PRT_signal_omrsig_register_os_handler_exiting Group=signal Overhead=1 Level=1 NoEnv Template="omrsig_register_os_handler: Exiting, rc=%d, portLibrarySignalFlag=0x%X, newOSHandler=%p, oldOSHandler=%p"
 TraceException=Trc_PRT_signal_omrsig_register_os_handler_invalid_portlibSignalFlag Group=signal Overhead=1 Level=1 NoEnv Template="omrsig_register_os_handler: ERROR, portlibSignalFlag=0x%X is either zero or has multiple signal bits set."
+
+TraceEntry=Trc_PRT_signal_omrsig_is_master_signal_handler_entered Group=signal Overhead=1 Level=1 NoEnv Template="omrsig_is_master_signal_handler: Entered, osHandler=%p"
```
Comment (pshipton) [inline port/common/omrport.tdf]: I don't think these need to be level 1 tracepoints.

File: port/common/omrport.tdf [hunk 2]
```diff
@@ -1069,3 +1069,6 @@ TraceException=Trc_PRT_signal_mapOSSignalToPortLib_ERROR_unknown_signal Group=si
...
+TraceExit=Trc_PRT_signal_omrsig_is_master_signal_handler_exiting Group=signal Overhead=1 Level=1 NoEnv Template="omrsig_is_master_signal_handler: Exiting, rc=%d"
```
Comment (pshipton) [inline port/common/omrport.tdf]: Probably want to end the file with a newline

File: port/common/omrport.tdf (hunk 1 above)
Comment (babsingh) [reply port/common/omrport.tdf]: changed to level 3 tracepoints.

File: port/common/omrport.tdf (hunk 2 above)
Comment (babsingh) [reply port/common/omrport.tdf]: added a newline.

**Pull request merged**